airflow/auth/managers/base_auth_manager.py

```diff
@@ -68,14 +68,8 @@ class BaseAuthManager(Generic[T], LoggingMixin):
     Class to derive in order to implement concrete auth managers.
 
     Auth managers are responsible for any user management related operation such as login, logout, authz, ...
-
-    :param appbuilder: the flask app builder
     """
 
-    def __init__(self, appbuilder: AirflowAppBuilder | None = None) -> None:
```

Comment:
Probably worth a newsfragment for this change in the authmanager interface.

Reply:
You are correct

Reply:
I added a newsfragment `aip-79.significant.rst`. The used the AIP naming because I am sure there will other breaking changes in the auth manager interface and I think it is better to aggregate them in one single newsfragment.